### PR TITLE
use .ogg files with pitch changes

### PR DESCRIPTION
**Pull request by notalex-sh**

### CODE REVIEW

File: shared.js [hunk 1]
```diff
@@ -2,71 +2,72 @@
...
-    var step = NOTES.indexOf(note);
-    var power = Math.pow(2, (octave * 12 + step - 57)/12);
-    var pitch = 440 * power;
-    return pitch;
-}
-
-function playInstrumentNote(instrument, note) {
-    // Get the base note and the octave so that I can get the frequency to oscillate
+// Number of semitones each note is above C
+const noteToSemitones = {
```
Comment (robinchew) [inline shared.js:6]: Shouldnt have removed NOTES. If you want the index, you could have us|d `NOTES.indexOf('C#')`.

Comment (robinchew) [reply shared.js:6]: Ok I see that you need A# instead of Bb. So nevermind. I do want to keep the old code though, so is this replacing the code that synthetically create sound? i do want to keep that around

File: shared.js [hunk 2]
```diff
@@ -2,71 +2,72 @@
...
-// https://gist.github.com/marcgg/94e97def0e8694f906443ed5262e9cbb?permalink_comment_id=3896533#gistcomment-3896533
-var NOTES = ["C", "C#", "D", "Eb", "E", "F", "F#", "G", "G#", "A", "Bb", "B"];
-function getPitch(note, octave) {
-    var step = NOTES.indexOf(note);
-    var power = Math.pow(2, (octave * 12 + step - 57)/12);
-    var pitch = 440 * power;
-    return pitch;
-}
-
-function playInstrumentNote(instrument, note) {
```
Comment (robinchew) [inline shared.js:15]: Bring back the old playInstrumentNote function. Create new one called something like playInstrumentNoteFromFile.

**Comment by notalex-sh** [comment]
I have just addressed your comments:
- Added back the playInstrumentNote function
- Changed the name of the ogg note function to playInstrumentNoteFunction
- Made all the notes consisted changing Bd -> A#

File: shared.js [hunk 3]
```diff
@@ -82,42 +148,57 @@ const shared = () => {
   }
 
   function jsToCss(css, i = 0) {
```
Comment (robinchew) [inline shared.js:150]: i appreciate you are making the coding format more consistent, but it's hard to review. Stylistic changes should be a separate PR. just a note for next time

File: .DS_Store
Comment (robinchew) [inline .DS_Store:1]: remove this folder

Comment (robinchew) [reply .DS_Store:1]: ooos i merged before u removed this. I hope this folder isnt big, taking up unnecessary git space

File: shared.js (hunk 3 above)
Comment (notalex-sh) [reply shared.js:150]: This was the code you originally had to play the notes, I did not want to remove it so I left it in, I can remove this and the DS_store folder and reopen a PR for this branch if you would like?

Comment (robinchew) [reply shared.js:150]: i mean look at the Files Changed tab. https://github.com/robinchew/music/pull/3/files

There is so many lines changed but they are really just indentation. which is good but not good of im trying read new code.
